### PR TITLE
fix(kit): `InputDateTime` should format textfield value on blur 

### PR DESCRIPTION
**Pull request by alyonagerasimova**
Closes https://github.com/taiga-family/taiga-ui/issues/7638

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/kit/components/input-date-time/input-date-time.component.ts [hunk 1]
```diff
@@ -312,7 +312,7 @@ export class TuiInputDateTimeComponent
         if (
             this.value[0] === null ||
             this.value[1] !== null ||
-            this.nativeValue.length <= this.fillerLength + DATE_TIME_SEPARATOR.length ||
+            this.nativeValue.length === this.fillerLength ||
```
Comment (nsbarsukov) [inline projects/kit/components/input-date-time/input-date-time.component.ts:315]: @alyonagerasimova 
Thanks for your contribution!

* Could you add playwright tests for this case inside [this file](https://github.com/taiga-family/taiga-ui/blob/v3.x/projects/demo-playwright/tests/kit/input-date-time/input-date-time.spec.ts), please ?
  ```ts
  test('Good description', async ({page}) => {
      await tuiGoto(page, '/components/input-date-time/API?timeMode=HH:MM:SS.MSS');
  
      await inputDateTime.textfield.fill('07.06.2024, 23:59');
      await inputDateTime.textfield.blur();
  
      await expect(inputDateTime.textfield).toHaveValue('07.06.2024, 23:59:00.000');
  });
  ```

* Cherry-pick your fix to `main` branch too, please

Comment (nsbarsukov) [reply projects/kit/components/input-date-time/input-date-time.component.ts:315]: I see you close your previous PR to `main`:
* https://github.com/taiga-family/taiga-ui/pull/7656

Sorry for confusion.
Let's me clarify it.
We require this fix in both `v3.x` and `main` branches (2 PR in total).

Comment (alyonagerasimova) [reply projects/kit/components/input-date-time/input-date-time.component.ts:315]: Yes, I prematurely deleted my first fork repository in order to fork it again. I will open a new PR in the main branch

Comment (alyonagerasimova) [reply projects/kit/components/input-date-time/input-date-time.component.ts:315]: https://github.com/taiga-family/taiga-ui/pull/7669

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**